libzip.mk

```diff
@@ -0,0 +1,56 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS          += libzip
+DOWNLOAD             += https://libzip.org/download/libzip-$(LIBZIP_VERSION).tar.gz
+LIBZIP_VERSION       := 1.7.0
+DEB_LIBZIP_V         ?= $(LIBZIP_VERSION)
+
+libzip-setup: setup
+	$(call EXTRACT_TAR,libzip-$(LIBZIP_VERSION).tar.gz,libzip-$(LIBZIP_VERSION),libzip)
+
+ifneq ($(wildcard $(BUILD_WORK)/libzip/.build_complete),)
+libzip:
+	@echo "Using previously built libzip."
+else
+libzip: libzip-setup
+	cd $(BUILD_WORK)/libzip && cmake . \
+		-DCMAKE_BUILD_TYPE=Release \
+		-DCMAKE_SYSTEM_NAME=Darwin \
+		-DCMAKE_CROSSCOMPILING=true \
+		-DCMAKE_INSTALL_NAME_TOOL=$(I_N_T) \
+		-DCMAKE_INSTALL_PREFIX=/ \
+		-DCMAKE_INSTALL_NAME_DIR=/usr/lib \
+		-DCMAKE_INSTALL_RPATH=/usr \
+		-DCMAKE_OSX_SYSROOT="$(SYSROOT)" \
+		-DCMAKE_C_FLAGS="$(CFLAGS)" \
+		-DCMAKE_CXX_FLAGS="$(CXXFLAGS)" \
+		-DCMAKE_SHARED_LINKER_FLAGS="-L$(BUILD_BASE)/usr/lib" \
+		-DCURRENT_VENDOR=debian \
```

Comment:
You likely don't need current vendor or any options like that; That's an apt specific option. Vet these options and deduce which ones aren't necessary.